_includes/layout.tsx

```diff
@@ -2,7 +2,7 @@ export default function Layout(props: Lume.Data) {
   const reference = props.url.startsWith("/api");
 
   return (
-    <html lang="en" class={reference ? "" : "h-dvh overflow-hidden"}>
```

Comment:
Overflow on parents prevents child elements from being able to use `position: sticky`. When we encounter an overflow issue we should address that instead of using a sledge hammer on a parent element.